docs/api/data-view-objects.md

```diff
@@ -103,4 +104,44 @@ let objects: powerbi.DataViewObjects = {
 DataViewObjects.getFillColor(objects, property);
 
 // returns: yellow
-```
\ No newline at end of file
+```
+
+## getCommonValue
+
+This function is universal function in order to get color or value of the given object.
```

Comment:
This function is a universal function for retrieving the color or value of a given object.